src/decoration/decoration.ts

```diff
@@ -21,22 +24,23 @@ const decorationType = () : vscode.TextEditorDecorationType => {
  */
 const decorationOption = (has_error : boolean, has_annotation : boolean) : vscode.TextEditorDecorationType => {
     //print
-    
     return vscode.window.createTextEditorDecorationType({
         // decoration options for dark mode
         // ---- NOTE: THE TEXT UNDERLINES FOR ANNOTATED NOTES WILL BE MADE BETTER IN FUTURE VERSIONS. FOR NOW, THEY WILL EXIST AS PROOF OF CONCEPT ----
         dark: {
-            backgroundColor: 
-                has_error ? getConfiguration().decorationColors?.error :
-                getConfiguration().decorationColors?.dark,
-            textDecoration: has_annotation ? '#EE59FF wavy underline' : ''
+            textDecoration: 
+                has_error ? getConfiguration().decorationColors?.error + " wavy underline":
```

Comment:
Is there no added decoration if the has_error conditional fails? This should highlight the code in something light, or underline it in a light color to indicate there is a term at that location in code.

Reply:
That is how it currently works, though I can make a quick change to fix that.